src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -559,18 +652,22 @@ function App() {
 <Block name="TypeScript">
```

Comment:
The block name should be "React Native" correct?

Reply:
I dont think so, there is a react-native Inline filter for the platform, and it has 2 blocks TypeScript and JavaScript